src/protagonist/Portal/Pages/Queue/Index.cshtml

```diff
@@ -88,4 +88,33 @@ else
         There are currently no batches here
     </div>
 }
-@await Component.InvokeAsync("Pager", new { values = Model.PagerValues })
\ No newline at end of file
+@await Component.InvokeAsync("Pager", new { values = Model.PagerValues })
+
+@section Scripts 
+{
+    <script>
+         $('.test-button').click(function() {
+            let batchId = $(this).data('batch');
+            let button = this;
```

Comment:
(This isn't worth fixing, just an FYI) `let button = $(this);` should allow you use `button` without needing to cast as jquery element each time (`$(button).text('Superseded');` => `button.text('Superseded');`)

Reply:
That's a good tip! Appreciated.